app/views/dashboard/index.html.erb

```diff
@@ -13,9 +13,11 @@
 </div>
 
 <h2>Items Ready To Ship</h2>
-<div id="merchant-items">
+  <!-- <div id="Items-Ready-To-Ship"> -->
```

Comment:
I saw that this was commented out, do we still need this section to be identified? We can nest the divs for our within blocks if we need to-- I just did it on my latest PR so take a peep if it would be helpful here!

Reply:
Woops .  Thanks for catching that.  I meant to delete it.